crates/driver/src/infra/solver/dto/auction.rs

```diff
@@ -275,39 +215,58 @@ pub struct Auction {
 #[serde_as]
 #[derive(Debug, Serialize)]
 #[serde(rename_all = "camelCase")]
-struct Order {
+pub struct Order {
```

Comment:
Suddenly making so many things public seems sketchy. Hopefully there is a way around that.